Signal/src/views/OWSSystemMessageCell.m

```diff
@@ -51,7 +51,10 @@ - (instancetype)init
 
 - (void)commontInit
 {
-    OWSAssert(!self.imageView);
```

Comment:
nit: Are we calling `commonInit` twice? If so it's probably not an `init` method. If we're not wittingly calling it twice, I prefer the (previous) failing asserts so we could fix a bug.